src/components/common/Actions.vue

```diff
@@ -79,12 +78,12 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Lega
 
   /** True if the Save button should be disabled. */
   private get isSaveButtonDisabled (): boolean {
-    return (!this.isEntityType || this.isBusySaving || this.isEditing)
```

Comment:
Instead of this, I conditionally rendered the buttons. It should have the same effect -- the user cannot use the buttons while the entity is loading.